test/runner.zig

```diff
@@ -208,33 +217,25 @@ fn mainTerminal() void {
 		testing.log_level = .warn;
 
 		const test_node = root_node.start(test_fn.name, 0);
-		if(!have_tty) {
-			std.debug.print("{d}/{d} {s}...", .{i + 1, test_fn_list.len, test_fn.name});
-		}
+
+		const padding_length: usize = max_length - test_fn.name.len + 3;
+		const padding = padding_buffer[0..padding_length];
+
+		std.debug.print("{d: >4}/{d: <4} {s}{s}", .{i + 1, test_fn_list.len, test_fn.name, padding});
 		is_fuzz_test = false;
 		if(test_fn.func()) |_| {
 			ok_count += 1;
 			test_node.end();
-			if(!have_tty) std.debug.print("OK\n", .{});
+			std.debug.print("OK\n", .{});
 		} else |err| switch(err) {
 			error.SkipZigTest => {
 				skip_count += 1;
-				if(have_tty) {
-					std.debug.print("{d}/{d} {s}...SKIP\n", .{i + 1, test_fn_list.len, test_fn.name});
-				} else {
-					std.debug.print("SKIP\n", .{});
-				}
+				std.debug.print("SKIP\n", .{});
 				test_node.end();
 			},
 			else => {
 				fail_count += 1;
-				if(have_tty) {
-					std.debug.print("{d}/{d} {s}...FAIL ({s})\n", .{
-						i + 1, test_fn_list.len, test_fn.name, @errorName(err),
-					});
-				} else {
-					std.debug.print("FAIL ({s})\n", .{@errorName(err)});
-				}
+				std.debug.print("FAIL ({s})\n", .{@errorName(err)});
```

Comment:
I would prefer if the error was in the next line
```suggestion
				std.debug.print("FAIL\n{s}:\n", .{@errorName(err)});
```